ansible/roles/operator-pipeline/tasks/community-release-pipeline-trigger.yml

```diff
@@ -114,6 +123,12 @@
                           resources:
                             requests:
                               storage: 100Mi
-                    - name: registry-credentials
```

Comment:
This secret is used by the community when submitting IIB build. Is this going to work when switching to a common operator pipeline?